cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/enums/ResellerPricingVisibility.java

```diff
@@ -0,0 +1,15 @@
+package com.cloudesire.platform.apiclient.dto.model.enums;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public enum ResellerPricingVisibility
+{
+    @ApiModelProperty( "Pricing is not visible to customer" )
```

Comment:
ProductVersion?

Reply:
"The pricing for the product version is not..."?

Reply:
non va bene, ne cambieresti il significato